.gitignore

```diff
@@ -46,3 +46,7 @@ rpm/src/
 
 *.pyc
 test/acceptance/properties.json
+cmake_install.cmake
+CMakeFiles
+Makefile
```

Comment:
I understand these Makefiles are generated inside `BUILD_*` directories, so line 5 in this file should avoid them. Thus, the entry at 51 is not necessary (it seems that the same for CMakeFiles and cmake_install.cmake).